calib/calibtool.h

```diff
@@ -30,12 +32,13 @@ typedef struct _calib_t {
 
 	/* utility related */
 	const char *(*help)(void);             /* help message description */
-	int (*interpret)(const char *, float); /* calibration file data interpreter */
 	int (*write)(FILE *);                  /* calibration file data write */
-} calib_t;
+
+	calib_t *(*calStructGet)(void); /* returns internal structure of libcalib:calib_t type */
```

Comment:
Can it return const value?
```suggestion
	calib_t *(*dataGet)(void); /* returns internal structure of libcalib:calib_t type */
```